docs/design/node-resource-reservation-design.md

```diff
@@ -0,0 +1,52 @@
+# Volcano node resource reservation
+## background
+* Consider such situation: there are thounsands of pods to be scheduled every day, in 1 to 2 o'clock 500 low priority pods are created and scheduled which used 99% of cluster resource, in 2 to 3 o'clock 10 pods which need to be reserved are created, however, low priority pods are still running, pods which need to be reserved can not be scheduled due to lack of resource.
+* Users want pods which need to be reserved in 2 to 3 o'clock have resource to schedule immediately every day and not preempt low priority pods because some low priority pods have already run many days.
+## design
+![annotation](images/node-resource-reservation-annotation.png) 
+### recognize pods which need to be reserved
+There are two ways to recognize these pods:
+* set annotation volcano.sh/reserveable: true in podgroup 
+which means all pods under podgroup are reserve tasks
+* set annotation volcano.sh/reserveable: true in pod
+which means this pod is reserve task
+### recognize pod max running time
+* set annotation volcano.sh/maximum-runtime: 500s in podgroup
+which means the podgroup will run for a maximum of 500 seconds
+* set annotation volcano.sh/maximum-runtime: 500s in pod
+which means this pod will run for a maximum of 500 seconds
+### reserve plugin
+#### configuration
+```
+- plugins:
+   - name: reserve
+     arguments:
+       reservelabels:
+       - nodeSelector:
+           business_type: ebook
+         startHour: 3
```

Comment:
Maybe we'd better use the Linux crontab format? Go has such packages to directly decode such format

Reply:
good idea, I have modified the doc